mods/xpanes/init.lua

```diff
@@ -132,11 +127,24 @@ function xpanes.register_pane(name, def)
 		})
 	end
 
-	def.on_construct = function(pos)
-		update_pane(pos, name)
-	end
-
-	minetest.register_node(":xpanes:"..name, def)
+	minetest.register_node(":xpanes:"..name, {
+		description = def.description,
+		inventory_image = def.inventory_image,
+		wield_image = def.wield_image,
+		drawtype = "airlike",
+		paramtype = "light",
+		is_ground_content = false,
+		sunlight_propagates = true,
+		walkable = false,
+		pointable = false,
+		diggable = false,
+		buildable_to = true,
+		air_equivalent = true,
+		groups = def.groups,
+		on_construct = function(pos)
+			update_pane(pos, name)
+		end
```

Comment:
This misses `sounds` and makes it inpossible to add any other custom fields.  Try adding fields to the definition instead of creating a seperate table.